github.ghactivity.php

```diff
@@ -289,4 +290,47 @@ private function get_repo_activity( $repo_name = '' ) {
 
 		return $this->get_github_data( $query_url );
 	}
+
+	/**
+	 * Get the list of repo label names from GH API
+	 *
+	 * @since 2.0.0
+	 *
+	 * @param string $repo_name Name of the repo we are interested in.
+	 *
+	 * @return array Array of repo label names.
+	 */
+	public function get_repo_label_names( $repo_name ) {
```

Comment:
Any reason we can't query those locally since we already got all the issues and their associated labels?

Reply:
That's possible. I actually tried to get them locally for the first try. But it turns out to be a tricky task, since we are not separating these labels per repo, so it would require loop through every label term and checking it's metadata.